packages/malloy-interfaces/thrift/malloy.thrift

```diff
@@ -277,10 +246,12 @@ struct Field {
   1: required Expression expression,
   // TODO only two kinds of distinguishable annotations are before `aggregate:` and before `name is value`
   // between `name` and `is`, or between `is` and `value` are converted to before `name`.
```

Comment:
Are we okay with not distinguishing between the various positions annotations on a definition can go?

Reply:
Michael would like to know whether the annotation was from the group or object (or inheritance), but doesn't care whether it was before the is or after the is.